modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/js/configuration/product_list_renderer.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function ({namespace}) {
+	const form = document.getElementById(`${namespace}fm`);
+	const select = document.getElementById(
+		`${namespace}cpContentListRendererKeySelect`
```

Comment:
Is the namespace needed here?

Reply:
Yes. select tag is `<select class="form-control" id="_com_liferay_portlet_configuration_web_portlet_PortletConfigurationPortlet_cpContentListRendererKeySelect" name="_com_liferay_portlet_configuration_web_portlet_PortletConfigurationPortlet_preferences--cpContentListRendererKey--">`

Reply:
Yes. select tag is `<select class="form-control" id="_com_liferay_portlet_configuration_web_portlet_PortletConfigurationPortlet_cpContentListRendererKeySelect" name="_com_liferay_portlet_configuration_web_portlet_PortletConfigurationPortlet_preferences--cpContentListRendererKey--">`

Reply:
Ok, perfect. I asked because I didn't see the namespace prefix [here](https://github.com/liferay-commerce/liferay-portal/pull/2521/commits/38193312c4ef7fa4126a6acd370a312ae8015c1f#diff-51e94aac299cf2e6bbc8fd9d5a5e374a2f5226e91cdce4b82f566b54e34b3b2bR26)